test/e2e/suit_test.go

```diff
@@ -128,7 +128,12 @@ var _ = BeforeSuite(func() {
 	chart, err := loader.Load(chartPath)
 	Expect(err).NotTo(HaveOccurred())
 	Expect(chart).NotTo(BeNil())
-	release, err := installAction.Run(chart, nil)
+
+	values := make(map[string]interface{})
+	if imageTag := os.Getenv("IMAGE_TAG"); imageTag != "" {
+		values["image.tag"] = imageTag
+	}
+	release, err := installAction.Run(chart, values)
```

Comment:
We could use values defined in `charts/spark-operator-chart/ci/ci-values.yaml` as it is aimed to be used in CI workflow.

Reply:
Using the `chartutils` package to load these values now